package.json

```diff
@@ -25,6 +25,8 @@
     "test:cli": "cross-env KNEX_PATH=../knex.js KNEX=bin/cli.js jake -f test/jake/Jakefile",
     "db:start": "docker-compose -f scripts/docker-compose.yml up --build -d mysql oracledbxe postgres mssql; docker-compose -f scripts/docker-compose.yml up initmssqlknexdb waitmysql waitpostgres waitoracledbxe",
     "db:stop": "docker-compose -f scripts/docker-compose.yml down",
+    "db:start:dev": "docker-compose -f scripts/docker-compose-dev.yml up -d",
```

Comment:
I suppose this does the same thing without an additional docker-compose file
```
docker-compose -f scripts/docker-compose.yml up --build -d postgres && docker-compose -f scripts/docker-compose.yml up waitpostgres
```

Also naming could be db:start:postgres

Reply:
I get this error when I try to run your command:
```
Creating scripts_waitpostgres_1 ... done
Attaching to scripts_waitpostgres_1
waitpostgres_1     | psql: error: could not connect to server: could not connect to server: Connection refused
waitpostgres_1     |    Is the server running on host "postgres" (172.19.0.2) and accepting
waitpostgres_1     |    TCP/IP connections on port 5432?
waitpostgres_1     |  ?column?
waitpostgres_1     | ----------
waitpostgres_1     |         1
waitpostgres_1     | (1 row)
waitpostgres_1     |
scripts_waitpostgres_1 exited with code 0
```

Any ideas?

Reply:
Also `docker-compose -f scripts/docker-compose.yml down postgres && docker-compose -f scripts/docker-compose.yml down waitpostgres` doesn't work, what would be the right way to shut it down?

Reply:
actually I don't think I have ever tried to shutdown only a single service.... maybe docker-compose stop postgres is enough?

Reply:
or even `docker-compose -f scripts/docker-compose.yml rm -fs postgres` which stops and removes the container.

Reply:
apparently docker-compose is smart enough, stopping whole script manages to identify what is running and stop just that.